### PR TITLE
use a string for the resource migration file

### PR DESCRIPTION
**Pull request by ghedamat**
we assume that the migration file in the entity is a string and we internally cast it to a resource

### CODE REVIEW

File: test/crustacean/t_migrations.clj [hunk 1]
```diff
@@ -30,20 +30,20 @@
   (get-migrations ..entity..) => ..migrations..
   (provided
     ..entity.. =contains=> {:migration-file ..somefile..}
-    (read-string (slurp ..somefile..)) => ..migrations..)
+    (clojure.java.io/resource ..somefile..) => ..some-resource..
```
Comment (ghedamat) [inline test/crustacean/t_migrations.clj:33]: @mveytsman kinda hate stubbing so much

**Comment by ghedamat** [comment]
this is clearly leaking the internal of knowing that "resource/" is appended to the path,

definitely needs refactoring but it's green for now and helps me move forward

**Comment by mveytsman** [comment]
Can you bump the crustacean version for this, since it's breaking the API?

Otherwise, looks great :)

**Comment by ghedamat** [comment]
@mveytsman done

**Pull request merged**